lib/index.js

```diff
@@ -213,6 +213,8 @@ function getHeaders (uri, auth, opts) {
     'user-agent': opts.userAgent,
   }, opts.headers || {})
 
+  headers['npm-use-webauthn'] = opts.authType === 'webauthn'
```

Comment:
It seems like it would be more worthwhile just to pass along whatever the `authType` is & let the configured registry handle that value vs. create individual headers for all of the types.

```suggestion
  headers['npm-auth-type'] = opts.authType
```

Reply:
Done: https://github.com/npm/npm-registry-fetch/pull/123/commits/5038cf101b7f77acb2c6214e5c741124337039f1

Hope we can get all the changes we need to do elsewhere in the CLI reviewed and merged quickly then 🙂